lint/unnamedResult_checker.go

```diff
@@ -20,9 +20,21 @@ func init() {
 
 type unnamedResultChecker struct {
 	checkerBase
+
+	onlyExported bool
+}
+
+func (c *unnamedResultChecker) Init() {
+	v, has := c.ctx.params["onlyExported"]
+	if flag, ok := v.(bool); has && ok {
+		c.onlyExported = flag
+	}
```

Comment:
I think this can be simplified to `c.onlyExported = c.ctx.params["onlyExported"]` since default values are the same.

Reply:
Hmm.. Maybe this is an idea for checker, too? ;)